plugin-server/src/main/pluginsServer.ts

```diff
@@ -238,6 +246,14 @@ export async function startPluginsServer(
         serverInstance.queue = queue
         serverInstance.stop = closeJobs
 
+        if (!(determineNodeEnv() === NodeEnv.Test)) {
+            // start http server used for the healthcheck
+            const httpServer = createHttpServer(hub, serverConfig)
+            httpServer.listen(HTTP_SERVER_PORT, () => {
+                status.info('🩺', `Status server listening on port ${HTTP_SERVER_PORT}`)
+            })
+        }
+
```

Comment:
I would move the `.listen` call also into `createHttpServer`, so that `pluginServer.ts` just knows enough to start and stop it. Otherwise, this is a perfectly fine spot.